sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample1_BackupHelloWorldAsync.md

```diff
@@ -53,7 +53,7 @@ Alternatively, it is possible to [generate a SAS token in Storage Explorer](http
 RestoreOperation restoreOperation = await Client.StartRestoreAsync(backupFolderUri, sasToken);
 
 // Wait for completion of the RestoreOperation.
-Response restoreResult = await restoreOperation.WaitForCompletionAsync();
+RestoreResult restoreResult = await restoreOperation.WaitForCompletionAsync();
```

Comment:
Nit: while you're at it, we shouldn't use the implicit cast operator for samples. So while you're in here, maybe use `Response<RestoreResult>` while you're at it (or later...whichever).